homeassistant/components/light/hue.py

```diff
@@ -54,6 +54,10 @@ def _find_host_from_config(hass, filename=PHUE_CONFIG_FILE):
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """ Gets the Hue lights. """
+    if config.get('ignore_unreachable', False):
+        _CONFIGURING['ignore_unreachable'] = True
```

Comment:
Why store this in the `_CONFIGURING` dict?

Also, you can simplify these 4 lines easily:

```python
_CONFIGURING['ignore_unreachable'] = config.get('ignore_unreachable', False) 
```